core/templates/pages/exploration-editor-page/suggestion-modal-for-editor-view/exploration-editor-suggestion-modal.controller.spec.ts

```diff
@@ -68,35 +68,35 @@ describe('Exploration Editor Suggestion Modal Controller', function() {
       });
     }));
 
-    it('should check properties set after controller is initialized',
-      function() {
-        expect($scope.isNotHandled).toEqual(false);
-        expect($scope.canEdit).toBe(true);
-        expect($scope.commitMessage).toBe('');
-        expect($scope.reviewMessage).toBe('');
-        expect($scope.canReject).toBe(false);
-        expect($scope.canAccept).toBe(false);
-        expect($scope.currentContent).toBe(currentContent);
-        expect($scope.newContent).toBe(newContent);
-        expect($scope.errorMessage).toBe(
-          'This suggestion has already been rejected.');
-      });
+    it('should initialize correctly $scope properties after controller' +
+      ' initialization', function() {
+      expect($scope.isNotHandled).toEqual(false);
+      expect($scope.canEdit).toBe(true);
+      expect($scope.commitMessage).toBe('');
+      expect($scope.reviewMessage).toBe('');
+      expect($scope.canReject).toBe(false);
+      expect($scope.canAccept).toBe(false);
+      expect($scope.currentContent).toBe(currentContent);
+      expect($scope.newContent).toBe(newContent);
+      expect($scope.errorMessage).toBe(
+        'This suggestion has already been rejected.');
+    });
 
-    it('should accept suggestion', function() {
+    it('should accept suggestion from suggestion modal service', function() {
       spyOn(SuggestionModalService, 'acceptSuggestion').and.callThrough();
       $scope.acceptSuggestion();
 
       expect(SuggestionModalService.acceptSuggestion).toHaveBeenCalled();
     });
 
-    it('should reject suggestion', function() {
+    it('should reject suggestion from suggestion modal service', function() {
       spyOn(SuggestionModalService, 'rejectSuggestion').and.callThrough();
       $scope.rejectSuggestion();
 
       expect(SuggestionModalService.rejectSuggestion).toHaveBeenCalled();
     });
 
-    it('should cancel review', function() {
+    it('should cancel review from suggestion modal service', function() {
```

Comment:
from suggestion modal service? Make the name less dependent on the code (and the rest of the names in this file)

Reply:
Done.